src/evaluation/stats.jl

```diff
@@ -0,0 +1,142 @@
+using DrWatson
+using BSON
+using FileIO
+using Random
+using DataFrames
+using Statistics
+using StatsBase
+using EvalMetrics
+
+# metric names and settings 
+const BASE_METRICS = ["auc", "auprc", "tpr_5", "f1_5"]
+const PAT_METRICS = ["pat_1", "pat_5", "pat_10", "pat_20"]
+
+"""
+	_prefix_symbol(prefix, s)
+
+Modifies symbol `s` by adding `prefix` with underscore.
+"""
+_prefix_symbol(prefix, s) = Symbol("$(prefix)_$(s)")
+
+
+"""
+	_precision_at(p, scores, labels)
+
+Computes precision on portion `p` samples with highest score.
+"""
+function _precision_at(p, labels, scores)
+	pN = Int(round(floor(p*length(labels)))) # round is more forgiving
```

Comment:
Here you can do just `pN = floor(Int, p*length(labels))`

Reply:
Thanks for spotting this. It is an artifact from when I was playing with both floor and round options.